test/unit/kubernetes-deploy/kubernetes_resource_test.rb

```diff
@@ -0,0 +1,7 @@
+require 'test_helper'
+
+class KubernetesResourceTest < KubernetesDeploy::TestCase
+  def test_service_and_deployment_timeouts_are_equal
+    assert_equal KubernetesDeploy::Service.timeout, KubernetesDeploy::Deployment.timeout
```

Comment:
Suggestion: make the failure message explain why this is a good idea (I'm not sure it is obvious)